src/app/settings/settings.component.html

```diff
@@ -27,17 +27,19 @@
     </ion-item>
     <app-contact-number-form [page]="'settings'"></app-contact-number-form>
 
-    <ion-item *ngIf="isInMultiplePrograms()" lines="none" class="exp-switch" tabindex="0">
-      <ion-label class="subtitle-2 gray-2">Switch to another experience</ion-label>
-      <ion-button class="practera-btn mobile-btn" slot="end" (click)="switchProgram($event)" role="button" (keydown)="switchProgram($event)">SWITCH</ion-button>
-    </ion-item>
+    <ng-container *ngIf="mode !== 'embed'">
+      <ion-item *ngIf="isInMultiplePrograms()" lines="none" class="exp-switch" tabindex="0">
+        <ion-label class="subtitle-2 gray-2">Switch to another experience</ion-label>
+        <ion-button class="practera-btn mobile-btn" slot="end" (click)="switchProgram($event)" role="button" (keydown)="switchProgram($event)">SWITCH</ion-button>
+      </ion-item>
 
-    <div class="ion-margin-top support-mobile subtitle-1">
-      <ng-container *ngTemplateOutlet="supportContentTpl"></ng-container>
-    </div>
+      <div class="ion-margin-top support-mobile subtitle-1">
+        <ng-container *ngTemplateOutlet="supportContentTpl"></ng-container>
+      </div>
+    </ng-container>
   </ng-container>
 
-  <ng-container *ngIf="!utils.isMobile()">
+  <ng-container *ngIf="!utils.isMobile() && mode !== 'embed'">
```

Comment:
@shawnm0705 this is good enough. just a note, next time we can shorten and reduce logic on template by doing something similar using a function in the component.ts 
https://github.com/intersective/app-v2/blob/develop/src/app/assessment/assessment.component.ts#L110-L112